public/app/tasks/task-list.controller.js

```diff
@@ -47,7 +48,8 @@
 
             return TaskService.create(self.newTask)
                 .then(function (task) {
-                    self.tasks.push(task);
+                    //                    self.tasks.push(task);
```

Comment:
The previous version was better. You should not update whole list of tasks after creating new one